repoman/pym/repoman/modules/linechecks/whitespace/blank.py

```diff
@@ -0,0 +1,25 @@
+
+import re
+
+from repoman.modules.linechecks.base import LineCheck
+
+
+class EbuildBlankLine(LineCheck):
+	repoman_check_name = 'ebuild.minorsyn'
+	ignore_comment = False
+	blank_line = re.compile(r'^$')
+
+	def new(self, pkg):
+		self.line_is_blank = False
+
+	def check(self, num, line):
+		if self.line_is_blank and self.blank_line.match(line):
+			return 'Useless blank line on line: %d'
+		if self.blank_line.match(line):
+			self.line_is_blank = True
+		else:
```

Comment:
Is this needed? Should be false by default, right?

Reply:
yeah, it should be.  But I didn't modify this code, just relocated it from the original checks.py from before the linechecks split.  But to be honest, I'm not sure how this class check is iterated as to whether it is needed or not.

leaving for the time being...